RUFAS/routines/animal/pen.py

```diff
@@ -325,6 +325,21 @@ def is_populated(self) -> bool:
         """
         return len(self.animals_in_pen) > 0
 
+    @property
+    def needs_ration_formulation(self) -> bool:
+        """
+        Returns whether pen needs a ration formulated.
+
+        This is currently written to cover the case in which a ration was not formulated due to the pen being empty,
+         and on subsequent days is populated.
+
+        Returns
+        -------
+        bool
+            True if pen needs ration formulation.
```

Comment:
```suggestion
        Returns whether pen needs a ration formulated. This covers the case in which a ration was
        not formulated due to the pen being empty, but was populated in subsequent days.
```
I think the returns section isn't necessary, the method name is really clear and the first line of the docstring covers it well.